remote_templates/commit0.example.yml

```diff
@@ -0,0 +1,24 @@
+name: hello-world
+
+# Context will populated automatically or could be added manually
+context: 
+  awsCognito: 
+    poolID: 123
+    clientID: ABC
+
+packages: 
+  - source: './remote_templates/infrastructure'
+    params: 
+      apply: true
+
+  - source: './remote_templates/frontend'
+    params: 
+      ci: github
+
+  - source: './remote_templates/services'
+    # alternatively we can recursively support sub packges
+    packages: 
+      - './remote_templates/ci/go'
```

Comment:
Highlighting two different ways of dealing with including CI. 
1. as a package parameter that triggers a certain files to be included
2. as subpackages with the subpackage root path set to the parent's directory.

I prefer supporting sub-packages recursively, it encourages more reuse and separation of concerns.